src/GlobalDataStore.lua

```diff
@@ -113,4 +113,25 @@ function GlobalDataStore:UpdateAsync(key: string, transform)
 	return copyDeep(transformed), self.keyInfos[key]
 end
 
+function GlobalDataStore:RemoveAsync(key: string)
+	validateString("key", key, Constants.MAX_KEY_LENGTH)
+
+	if self.errors ~= nil then
+		self.errors:simulateError("RemoveAsync")
+	end
+
+	self.budget:yieldForBudget({ Enum.DataStoreRequestType.SetIncrementAsync })
+	self.yield:yield()
```

Comment:
Let's yield after getting the old value to be consistent with the other functions.

Reply:
fixed in 2nd commit